spec/factories/campaign.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :campaign do
+    title Faker::Book.title
```

Comment:
Since title must be unique I suggest using FactoryBot [sequences](http://www.rubydoc.info/gems/factory_bot/file/GETTING_STARTED.md#Sequences)
Using random data might cause some flakiness of the tests